### DISCUSSION TITLE
Changelog links for Gitlab commits and compare are broken

### DISCUSSION THREAD

**Discussion by markpakeltis** [projen/projen#2698] [Q&A] [answered]
Due to GItlab.com changed routes with dash ("-") like:
before: 
https://gitlab.com/my-group/project1/commit/5a38d1ef0affa92c2b80b7852e1c3xxxxxxxxxxxxx
now:
https://gitlab.com/my-group/project1/-/commit/5a38d1ef0affa92c2b80b7852e1c3xxxxxxxxxxxxx

Changelog links which are generated by projen not working anymore, they are generated without dash and by clicking them receiving 404 page not found.

Please adjust code to support the latest gitlab version (routes with dashes)

**Comment by mrgrain** [comment]
projen uses [standard-version](https://github.com/conventional-changelog/standard-version) to generate changelog notes but you can configure it easily using `versionrcOptions`.

But we don't have out-of-the-box Gitlab support so you'll have to do this yourself for now:
https://github.com/conventional-changelog/standard-version/blob/017dcb689587ed5abec2096be88ae8b052523f94/README.md?plain=1#L114-L120

**Comment by markpakeltis** [comment, marked as answer]
Thank you for the link! This was exactly what I also look for in first place. These what I added which works fine
```
const project = new javascript.NodeProject({
  // other code
  versionrcOptions: {
    commitUrlFormat: "{{host}}/{{owner}}/{{repository}}/-/commit/{{hash}}",
    compareUrlFormat: "{{host}}/{{owner}}/{{repository}}/-/compare/{{previousTag}}...{{currentTag}}",
    issueUrlFormat: "{{host}}/{{owner}}/{{repository}}/-/issues/{{id}}",
  },
 // other code
});
```
